localflavor/ro/forms.py

```diff
@@ -73,26 +75,36 @@ def __init__(self, max_length=13, min_length=13, *args, **kwargs):
     def clean(self, value):
         """
         CNP validations
+
+        Args:
+            value: the CNP code
         """
         value = super(ROCNPField, self).clean(value)
         if value in EMPTY_VALUES:
             return ''
-        # check birthdate digits
+
+        # check birthday digits
```

Comment:
`birthdate` is correct if the check includes the year. 

https://english.stackexchange.com/a/210491